CONTRIBUTING.md

```diff
@@ -1,269 +1,70 @@
-
-Whether a pull request is merged into Exonum Core rests with the project merge
-maintainers and ultimately the project lead.
-
-Maintainers will take into consideration if a patch is in line with the general
-principles of the project; meets the minimum standards for inclusion;
-and will judge the general consensus of contributors.
-
-In general, all pull requests must:
+## Finding something to fix or improve
 
-- have a clear use case, fix a demonstrable bug or serve the greater good
-    of the project (for example refactoring for modularisation);
-- be well peer reviewed;
-- have unit tests and sandbox tests where appropriate;
-- follow code style guidelines;
-- not break the existing test suite;
-- where bugs are fixed, where possible, there should be unit tests demonstrating
-    the bug and also proving the fix. This helps prevent regression.
+The [good first issue] label can be used to find the easy issues. If you want
+to work on an issue, please leave a comment so that we can assign it to you.
 
-### Peer Review
+## Linters
 
-Anyone may participate in peer review which is expressed by comments
-in the pull request. Typically reviewers will review the code for obvious errors,
-as well as test out the patch set and opine on the technical merits of the patch.
-Project maintainers take into account the peer review when determining
-if there is consensus to merge a pull request.
-The following language is used within pull-request comments:
+Notice that the repository uses a set of linters and static analysis:
 
-- LGTM means "Looks Good To Me";
-- Concept LGTM means "I agree in the general principle of this pull request".
+- [clippy]
+- [cargo audit]
+- [cargo-deadlinks]
+- [cargo-outdated]
+- [rustfmt 0.9.0]
+- [cspell]
+- [markdownlint-cli]
 
-Reviewers should include the commit hash which they reviewed in their comments.
+You can set up and run these tools locally (see [Travis script] for the details).
 
-Project maintainers reserve the right to weigh the opinions of peer reviewers
-using common sense judgement and also may weight based on meritocracy:
-Those that have demonstrated a deeper commitment and understanding towards
-the project (over time) or have clear domain expertise may naturally
-have more weight, as one would expect in all walks of life.
+## Conventions
 
-Where a patch set affects consensus critical code, the bar will be set
-much higher in terms of discussion and peer review requirements,
-keeping in mind that mistakes could be very costly to the wider community.
-This includes refactoring of consensus critical code.
+Generally, we follow common best practices established in the Rust community,
+but we have several additional conventions:
 
-## Release Policy
+- Don't use `debug!` log level.
 
-The project leader is the release manager for each Exonum Core release.
+  It is convenient to use `debug!` when you develop some feature and are only
+  interested in your logging output.
 
-## Copyright
+- ?
```

Comment:
The conventions section does not seem very helpful, because the most important conventions are enforced by CI anyway. I suggest just to drop this section 😆

Reply:
I cannot agree here. :smile: It happens relatively often for us when a new team member becomes aware of some "rules" only on the review stage. We need to document such conventions somewhere, so why not here?

Reply:
Sgtm!